packages/@okta/vuepress-site/docs/guides/terraform-enable-org-access/main/index.md

```diff
@@ -135,13 +165,14 @@ Create a Terraform configuration that uses the credentials that you created earl
 
    In the previous code sample, add your values to the following fields:
 
-   * `org_name`: Your Okta org name. For example, `exampleOrgName` from the full domain `https://exampleOrgName.okta.com`.
-   * `base_url`: Your Okta org domain
-   * `client_id`: The client ID of the service app that you created in an earlier step.
+   * `org_name`: Your Okta org name, which is the first part of your organization's Okta domain before `.okta.com`, `.oktapreview.com`, or `.okta-emea.com`. For example, if your Okta domain is `example-org.oktapreview.com`, the org name is `example-org`. Don't include the `-admin` suffix, which corresponds to your Admin Console URL. To confirm your Okta domain in the Admin Console, click your username in the upper-right corner and look for the text below your email address. An Okta Developer org has an org name with the form `dev-<number>`, such as `dev-000001`.
+   * `base_url`: Your Okta org domain. This is the end part of your domain URL: `.okta.com`, `.oktapreview.com`, or `.okta-emea.com`.
+   * `client_id`: The client ID of the API service app that you created. In the Admin Console, click **Applications**, then click the name of your API service app. In the app editor, click **General**. Under **Client Credentials**, copy the **Client ID** value.
    * `private_key`: Either the path to the private key file or the private key itself. Okta recommends storing the key in a separate location and using a secrets and encryption management system, such as HashiCorp Vault.
-   * `scopes`: A list of scopes required by the Terraform configuration. This example uses the `okta.groups.manage` scope.
+   * `scopes`: A list of API scopes that your Terraform code requests. This example creates a group, which requires the `okta.groups.manage` scope. 
```

Comment:
The definition reads passive voice... perhaps "The list of API scopes required by your Terraform configuration. The example below creates a group, which..."

Reply:
```suggestion
   * `scopes`: The list of API scopes required by your Terraform configuration. The example below creates a group, which requires the `okta.groups.manage` scope. 
```